.github/workflows/sanity-tests.yml

```
@@ -1,4 +1,4 @@
-name: UI Tests
+name: Upgrade CLI
```

```suggestion
name: Sanity Tests
```

We may use the name of the file at this level